posts/2021-05-11-database-ssl-primer.adoc

```diff
@@ -335,12 +348,11 @@ javax.net.ssl|DEBUG|41|Default Executor-thread-9|CertificateMessage.java:321|Pro
 )
 ----
```

Comment:
what is the suggested user action? How can they ensure the driver finds the certificate? 

Maybe:
"Check your configuration to ensure the certificate location is properly specified for the JDBC driver"
?

Reply:
I would say the resolution would be to:
"Check your server configuration and ensure that the keystore configured on the JDBC driver, or transportSecurity-1.0 exists and contains the certificate you expect the driver to send."

Reply:
updated